src/io/RootImporter.cc

```diff
@@ -142,7 +157,7 @@ RootImporter::load_physics_table_data()
         tables.push_back(a_table);
     }
 
-    return std::make_shared<std::vector<ImportPhysicsTable>>(std::move(tables));
+    return tables;
```

Comment:
This is the only object of the `RootImporter::result_type` struct that isn't a `shared_ptr`. Is this implementation the best way for other classes to access this data?

Reply:
Provisionally, yes. The shared `Params` data structures are in their final state and thus likely to be shared across multiple objects. The physics data is currently just a big blob of data suitable for handing off to a class (or classes) that can do something useful with it. A vector is appropriate for that situation, and we can adjust the return type later if we need to :)